slide-deck.js

```diff
@@ -292,7 +291,7 @@ class slideDeck extends HTMLElement {
   }
 
   // setup methods
-  #cleanString = (str) => str.trim().toLowerCase();
+  #cleanString = (str, lower = true) => str.trim().toLowerCase();
```

Comment:
I don't think you're using the `lower` argument- is that still needed?

Reply:
oops! 
```suggestion
  #cleanString = (str) => str.trim().toLowerCase();
```